common/setups/hybrid/util.py

```diff
@@ -0,0 +1,399 @@
+__all__ = [
+    "GmmDataInput",
+    "GmmInitArgs",
+    "GmmMonophoneArgs",
+    "GmmTriphoneArgs",
+    "GmmVtlnArgs",
+    "GmmSatArgs",
+    "GmmVtlnSatArgs",
+    "NnArgs",
+]
+
+import i6_core.meta as meta
+
+
+class GmmDataInput:
+    def __init__(
+        self,
+        corpus_object: meta.CorpusObject = None,
+        lexicon: dict = None,
+        lm: dict = None,
+        concurrent: int = 10,
+    ):
+        """
+        :param corpus_object: corpus_file: Path, audio_dir: Path, audio_format: str, duration: float
+        :param lexicon: file: Path, normalize_pronunciation: bool
+        :param lm: file: Path, type: str, scale: float
+        :param concurrent: concurrency for gmm hmm pipeline
+        """
+        self.corpus_object = corpus_object
+        self.lexicon = lexicon
+        self.lm = lm
+        self.concurrent = concurrent
+
+
+"""
+when getting an zero weights error, set:
+
+extra_config = sprint.SprintConfig()
+extra_config.allow_zero_weights = True
+{accumulate,split,align}_extra_args = {'extra_config': extra_config}
+
+'{accumulate,split,align}_extra_rqmt': {'mem': 10, 'time': 8},
+
+vtln align time = 8
+
+if not using the run function -> name and corpus almost always to be added
+"""
```

Comment:
What does this documentation belong to? Usually you would set it as a docstring to the module, some class or function.

And otherwise you would use comments (`#`) and not a string.